olclient/schemata/import.schema.json

```
@@ -159,6 +163,17 @@
 	  "examples": ["duc d'Otrante"]
 	}
       }
+    },
+    "contributors": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["name"],
+      "properties": {
+        "role": { "type": "string",
+        "examples": ["author", "editor", "illustrator"]},
+        "name": { "type": "string",
+        "examples": ["Hubbard, Freeman H.", "Joan Miró"]}
```

We generally would prefer names not in sort order ; I'm not sure if this endpoint cleans that up!

```suggestion
        "examples": ["Freeman H. Hubbard", "Joan Miró"]}
```